api/controllers/DashboardController.js

```diff
@@ -16,275 +14,360 @@ var dashboardController = {
+                            result.app_status = appRef[result.unique_app_id];
+                            result.tracking_ref =
+                                trackRef[result.unique_app_id];
                         }
-                    );
+                    }
+                }
+
+                var moment = require('moment');
+                var pageUpperLimit = offset + pageSize;
+                if (pageUpperLimit > resultCount) {
+                    pageUpperLimit = resultCount;
+                }
+                var totalPages =
+                    resultCount % pageSize === 0
+                        ? resultCount / pageSize
+                        : Math.floor(resultCount / pageSize) + 1;
+                var paginationMessage;
+                if (resultCount === 0) {
+                    paginationMessage = 'No applications found';
+                } else {
+                    paginationMessage =
+                        'Showing ' +
+                        (offset + 1) +
+                        ' &ndash; ' +
+                        pageUpperLimit +
+                        ' of ' +
+                        resultCount +
+                        ' applications submitted in the last 60 days';
+                }
+                const pageAttributes = {
+                    message: req.flash('info'),
+                    users_applications: results,
+                    moment,
+                    offset,
+                    sortOrder,
+                    paginationMessage,
+                    currentPage,
+                    totalPages,
+                    searchCriteria,
+                    user_data: userData,
+                    application_total: totalApplications,
+                };
+
+                return dashboardController._redirectToPage(pageAttributes, req, res);
+            }
+        );
+    },
 
-                })
-                .catch(function (error) {
-                    sails.log(error);
-                });
+    _redirectToPage(pageAttributes, req, res) {
+        const { electronicEnabled } =
+            pageAttributes.user_data.user;
+        let view = electronicEnabled
+            ? 'eApostilles/dashboard.ejs'
+            : 'dashboard.ejs';
+
+        if (req.query.ajax) {
+            view = electronicEnabled
+                ? 'partials/dashboardResultsEApp.ejs'
+                : 'partials/dashboardResults.ejs';
+            pageAttributes.layout = null;
+        }
 
-            //return res.view('dashboard.ejs', {users_applications: false, user:req.session.user});
-        });
-        });
+        return res.view(view, pageAttributes);
     },
+
     /**
      * @function openCoverSheet
      * @description Open the cover sheet
      * @param req {Array} - request object
      * @param res {Array} - response object
      * @return summary
      */
-    openCoverSheet: function(req, res) {
+    openCoverSheet: function (req, res) {
```

Comment:
this is bizarre (I know it's not your code!).. could this be changed to a normal method? 

```suggestion
    openCoverSheet (req, res) {
```

Reply:
I think it's the old school way of doing it, but it still works, although it does look weird